### DISCUSSION TITLE
[Bug]: Orange header (with score) is on top of object if object is small and next to upper edge of camera picture

### DISCUSSION THREAD

**Discussion by bonjour81** [blakeblackshear/frigate#15308] [Report a Bug] [answered]
### Checklist

- [X] I have updated to the latest available Frigate version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [ ] I have tried reproducing the issue in [incognito mode](https://www.computerworld.com/article/1719851/how-to-go-incognito-in-chrome-firefox-safari-and-edge.html) to rule out problems with any third party extensions or plugins I have installed.

### Describe the problem you are having

When going in frigate+ tab to review and submit some picture, in case tracked object is very small and very close to upper edge of the camera view, it may be hidden behind the orange header with score information.

See screenshot: 
![Capture d’écran du 2024-12-02 18-19-35](https://github.com/user-attachments/assets/1deb6a0f-9d95-4fe8-a91a-8070db5ef0fa)

maybe the orange header shall be move below the object in such conditions ?

Object can be seen fine in frigate+ account when reviewing, but still it make difficult to judge if we have to click green or red in we cannot seen the object before sending

### Steps to reproduce

1. go in frigate+ tab
2. open a snapshot with a very small object detected on top of the picture

### Version

0.14.1-f4f3cfa

### Frigate config file

```yaml
not related (I think)
```

### docker-compose file or Docker CLI command

```yaml
not related (I think)
```

### Relevant Frigate log output

```shell
not related (I think)
```

### Relevant go2rtc log output

```shell
not related (I think)
```

### Operating system

Debian

### Install method

Docker CLI

### Network connection

Wired

### Camera make and model

not related (I think)

### Screenshots of the Frigate UI's System metrics pages

not related (I think)

**Comment by hawkeye217** [comment, marked as answer]
This is fixed in https://github.com/blakeblackshear/frigate/pull/15310

Also worth mentioning that the next version of Frigate (0.15, currently in beta) has a feature called "Object Lifecycle", an improved way of quickly viewing objects without added drawings on them.

**Comment by bonjour81** [comment]
That was a quick fix 👍 thank you.
